src/lib/y2storage/proposal/autoinst_devices_creator.rb

```diff
@@ -295,8 +295,8 @@ def reuse_mds(reused_mds, previous_result)
       # @return                [Proposal::CreatorResult] Result containing the specified MD RAIDs
       def create_mds(mds, previous_result, devs_to_reuse)
         mds.reduce(previous_result) do |result, md|
-          devices = result.created_names { |d| d.respond_to?(:raid_name) && d.raid_name == md.name }
-          devices += devs_to_reuse.select { |d| d.raid_name == md.name }.map(&:reuse_name)
+          devices = result.created_names { |d| d.respond_to?(:raid_name) && md.name?(d.raid_name) }
```

Comment:
NP: as we were discussing, please, document here that `d.raid_name` is always equal to `md.name` when using the new AY format, or it is `/dev/md/X` otherwise.